docs/API_reference.md

```diff
@@ -275,6 +276,23 @@ There are currently two supported output formats: 'json' and 'dataframe'.
 The 'json' format simply returns the raw response, without page information and with multiple pages combined into a single data structure.  
 The 'dataframe' format transforms that data structure into a pandas DataFrame, which contains subparts in json, that can be split further by the user application.
 
+---
+
+##### plenty_api_get_property_selections: <a name='get-property-selections'></a>
+
+Return a mapping of Selection values for selction properties.
```

Comment:
s/Selection values/seleciton values/
s/selction/selection/

But actually, I would reword this to:
`Create a mapping of property IDs to the selection-values of the property with the ID of the selection and text values for all available languages.`